{"uid":"d6afe49143cac8db","name":"Переход на страницу с жалобами","fullName":"tests.test_excursium.TestUserCases#test_user_report_problem","historyId":"6ccf52d0675463b7f2ebb26dc5cf7f58","time":{"start":1753527739156,"stop":1753527754036,"duration":14880},"description":"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами","descriptionHtml":"<p>Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div/p/a[@href='/About/Contact#from-feedback']\"}\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\nStacktrace:\n#0 0x56055e75c87a <unknown>\n#1 0x56055e2012e0 <unknown>\n#2 0x56055e252e00 <unknown>\n#3 0x56055e252ff1 <unknown>\n#4 0x56055e2a1324 <unknown>\n#5 0x56055e2788dd <unknown>\n#6 0x56055e29e71c <unknown>\n#7 0x56055e278683 <unknown>\n#8 0x56055e244b5b <unknown>\n#9 0x56055e245f31 <unknown>\n#10 0x56055e7217cb <unknown>\n#11 0x56055e7255d4 <unknown>\n#12 0x56055e7082c9 <unknown>\n#13 0x56055e726178 <unknown>\n#14 0x56055e6ec6bf <unknown>\n#15 0x56055e749e78 <unknown>\n#16 0x56055e74a056 <unknown>\n#17 0x56055e75bb96 <unknown>\n#18 0x7f05f336d1f5 <unknown>","statusTrace":"self = <test_excursium.TestUserCases object at 0x7ff053e035c0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed4b42cd26d1c5c9e9adce49559d93a6\")>\n\n    @allure.title(\"Переход на страницу с жалобами\")\n    @allure.severity(\"Normal\")\n    @pytest.mark.user_case\n    def test_user_report_problem(self, driver):\n        \"\"\"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами\"\"\"\n        main_page = MainPage(driver)\n    \n        with allure.step(\"Открываем страницу\"):\n            driver.get(config.base_url)\n    \n        with allure.step(\"Переходим на страницу с экскурсиями\"):\n            driver.find_element(*main_page.all_excursion_btn).click()\n    \n        with allure.step(\"Находим и скролим до кнопки отправки жалобы\"):\n>           report_btn = driver.find_element(*main_page.report_btn)\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/test_excursium.py:116: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:922: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:454: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7ff053c1d450>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such element\",\"message\":\"no such element: Unable to locate element: {\\...\\\\n#16 0x56055e74a056 \\\\u003Cunknown>\\\\n#17 0x56055e75bb96 \\\\u003Cunknown>\\\\n#18 0x7f05f336d1f5 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div/p/a[@href='/About/Contact#from-feedback']\"}\nE         (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\nE       Stacktrace:\nE       #0 0x56055e75c87a <unknown>\nE       #1 0x56055e2012e0 <unknown>\nE       #2 0x56055e252e00 <unknown>\nE       #3 0x56055e252ff1 <unknown>\nE       #4 0x56055e2a1324 <unknown>\nE       #5 0x56055e2788dd <unknown>\nE       #6 0x56055e29e71c <unknown>\nE       #7 0x56055e278683 <unknown>\nE       #8 0x56055e244b5b <unknown>\nE       #9 0x56055e245f31 <unknown>\nE       #10 0x56055e7217cb <unknown>\nE       #11 0x56055e7255d4 <unknown>\nE       #12 0x56055e7082c9 <unknown>\nE       #13 0x56055e726178 <unknown>\nE       #14 0x56055e6ec6bf <unknown>\nE       #15 0x56055e749e78 <unknown>\nE       #16 0x56055e74a056 <unknown>\nE       #17 0x56055e75bb96 <unknown>\nE       #18 0x7f05f336d1f5 <unknown>\n\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: NoSuchElementException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1753527738877,"stop":1753527739156,"duration":279},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами","status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div/p/a[@href='/About/Contact#from-feedback']\"}\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\nStacktrace:\n#0 0x56055e75c87a <unknown>\n#1 0x56055e2012e0 <unknown>\n#2 0x56055e252e00 <unknown>\n#3 0x56055e252ff1 <unknown>\n#4 0x56055e2a1324 <unknown>\n#5 0x56055e2788dd <unknown>\n#6 0x56055e29e71c <unknown>\n#7 0x56055e278683 <unknown>\n#8 0x56055e244b5b <unknown>\n#9 0x56055e245f31 <unknown>\n#10 0x56055e7217cb <unknown>\n#11 0x56055e7255d4 <unknown>\n#12 0x56055e7082c9 <unknown>\n#13 0x56055e726178 <unknown>\n#14 0x56055e6ec6bf <unknown>\n#15 0x56055e749e78 <unknown>\n#16 0x56055e74a056 <unknown>\n#17 0x56055e75bb96 <unknown>\n#18 0x7f05f336d1f5 <unknown>","statusTrace":"self = <test_excursium.TestUserCases object at 0x7ff053e035c0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed4b42cd26d1c5c9e9adce49559d93a6\")>\n\n    @allure.title(\"Переход на страницу с жалобами\")\n    @allure.severity(\"Normal\")\n    @pytest.mark.user_case\n    def test_user_report_problem(self, driver):\n        \"\"\"Пользовательский сценарий перехода на вкладку с экскурсиями и на страницу с жалобами\"\"\"\n        main_page = MainPage(driver)\n    \n        with allure.step(\"Открываем страницу\"):\n            driver.get(config.base_url)\n    \n        with allure.step(\"Переходим на страницу с экскурсиями\"):\n            driver.find_element(*main_page.all_excursion_btn).click()\n    \n        with allure.step(\"Находим и скролим до кнопки отправки жалобы\"):\n>           report_btn = driver.find_element(*main_page.report_btn)\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/test_excursium.py:116: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:922: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:454: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7ff053c1d450>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such element\",\"message\":\"no such element: Unable to locate element: {\\...\\\\n#16 0x56055e74a056 \\\\u003Cunknown>\\\\n#17 0x56055e75bb96 \\\\u003Cunknown>\\\\n#18 0x7f05f336d1f5 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div/p/a[@href='/About/Contact#from-feedback']\"}\nE         (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\nE       Stacktrace:\nE       #0 0x56055e75c87a <unknown>\nE       #1 0x56055e2012e0 <unknown>\nE       #2 0x56055e252e00 <unknown>\nE       #3 0x56055e252ff1 <unknown>\nE       #4 0x56055e2a1324 <unknown>\nE       #5 0x56055e2788dd <unknown>\nE       #6 0x56055e29e71c <unknown>\nE       #7 0x56055e278683 <unknown>\nE       #8 0x56055e244b5b <unknown>\nE       #9 0x56055e245f31 <unknown>\nE       #10 0x56055e7217cb <unknown>\nE       #11 0x56055e7255d4 <unknown>\nE       #12 0x56055e7082c9 <unknown>\nE       #13 0x56055e726178 <unknown>\nE       #14 0x56055e6ec6bf <unknown>\nE       #15 0x56055e749e78 <unknown>\nE       #16 0x56055e74a056 <unknown>\nE       #17 0x56055e75bb96 <unknown>\nE       #18 0x7f05f336d1f5 <unknown>\n\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: NoSuchElementException","steps":[{"name":"Открываем страницу","time":{"start":1753527739157,"stop":1753527741310,"duration":2153},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Переходим на страницу с экскурсиями","time":{"start":1753527741310,"stop":1753527744015,"duration":2705},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Находим и скролим до кнопки отправки жалобы","time":{"start":1753527744016,"stop":1753527754035,"duration":10019},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div/p/a[@href='/About/Contact#from-feedback']\"}\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\nStacktrace:\n#0 0x56055e75c87a <unknown>\n#1 0x56055e2012e0 <unknown>\n#2 0x56055e252e00 <unknown>\n#3 0x56055e252ff1 <unknown>\n#4 0x56055e2a1324 <unknown>\n#5 0x56055e2788dd <unknown>\n#6 0x56055e29e71c <unknown>\n#7 0x56055e278683 <unknown>\n#8 0x56055e244b5b <unknown>\n#9 0x56055e245f31 <unknown>\n#10 0x56055e7217cb <unknown>\n#11 0x56055e7255d4 <unknown>\n#12 0x56055e7082c9 <unknown>\n#13 0x56055e726178 <unknown>\n#14 0x56055e6ec6bf <unknown>\n#15 0x56055e749e78 <unknown>\n#16 0x56055e74a056 <unknown>\n#17 0x56055e75bb96 <unknown>\n#18 0x7f05f336d1f5 <unknown>\n\n","statusTrace":"  File \"/app/tests/test_excursium.py\", line 116, in test_user_report_problem\n    report_btn = driver.find_element(*main_page.report_btn)\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 922, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"166e3e5f963ed8e3","name":"Скриншот при падении","source":"166e3e5f963ed8e3.png","type":"image/png","size":96847}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1753527754177,"stop":1753527754242,"duration":65},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"Normal"},{"name":"feature","value":"Пользовательские сценарии"},{"name":"tag","value":"user_case"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_excursium"},{"name":"subSuite","value":"TestUserCases"},{"name":"host","value":"4441db4abe9d"},{"name":"thread","value":"19-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_excursium"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":17,"skipped":0,"passed":36,"unknown":0,"total":54},"items":[{"uid":"b75a9ec50aa9befe","status":"passed","time":{"start":1753523652977,"stop":1753523659664,"duration":6687}},{"uid":"b566a501b615a2dd","status":"passed","time":{"start":1753523047218,"stop":1753523054994,"duration":7776}},{"uid":"38a220c22b62ea9","status":"passed","time":{"start":1753522714712,"stop":1753522720588,"duration":5876}},{"uid":"8c1cf10dd3d8dd57","status":"passed","time":{"start":1753521434252,"stop":1753521440626,"duration":6374}},{"uid":"43152a8e65b419c2","status":"broken","statusDetails":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element is not clickable at point (1247, 4242)\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\n#0 0x55b1621fc87a <unknown>\n#1 0x55b161ca12e0 <unknown>\n#2 0x55b161cf9c6c <unknown>\n#3 0x55b161cf7ad2 <unknown>\n#4 0x55b161cf5172 <unknown>\n#5 0x55b161cf4890 <unknown>\n#6 0x55b161ce70ea <unknown>\n#7 0x55b161d188b2 <unknown>\n#8 0x55b161ce6a6a <unknown>\n#9 0x55b161d18a7e <unknown>\n#10 0x55b161d3e71c <unknown>\n#11 0x55b161d18683 <unknown>\n#12 0x55b161ce4b5b <unknown>\n#13 0x55b161ce5f31 <unknown>\n#14 0x55b1621c17cb <unknown>\n#15 0x55b1621c55d4 <unknown>\n#16 0x55b1621a82c9 <unknown>\n#17 0x55b1621c6178 <unknown>\n#18 0x55b16218c6bf <unknown>\n#19 0x55b1621e9e78 <unknown>\n#20 0x55b1621ea056 <unknown>\n#21 0x55b1621fbb96 <unknown>\n#22 0x7f6a6df0b1f5 <unknown>","time":{"start":1753520106765,"stop":1753520111600,"duration":4835}},{"uid":"3182245c9089d170","status":"broken","statusDetails":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 893). Other element would receive the click: <div id=\"cookie-alert\" class=\"alert alert-primary py-2 m-0 bg-primary1 border-0 rounded-0 alert-dismissible fade show text-center overflow-hidden fixed-bottom\" role=\"alert\" style=\"display: block;\">...</div>\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\n#0 0x562a4f63687a <unknown>\n#1 0x562a4f0db2e0 <unknown>\n#2 0x562a4f133c6c <unknown>\n#3 0x562a4f131ad2 <unknown>\n#4 0x562a4f12f172 <unknown>\n#5 0x562a4f12e890 <unknown>\n#6 0x562a4f1210ea <unknown>\n#7 0x562a4f1528b2 <unknown>\n#8 0x562a4f120a6a <unknown>\n#9 0x562a4f152a7e <unknown>\n#10 0x562a4f17871c <unknown>\n#11 0x562a4f152683 <unknown>\n#12 0x562a4f11eb5b <unknown>\n#13 0x562a4f11ff31 <unknown>\n#14 0x562a4f5fb7cb <unknown>\n#15 0x562a4f5ff5d4 <unknown>\n#16 0x562a4f5e22c9 <unknown>\n#17 0x562a4f600178 <unknown>\n#18 0x562a4f5c66bf <unknown>\n#19 0x562a4f623e78 <unknown>\n#20 0x562a4f624056 <unknown>\n#21 0x562a4f635b96 <unknown>\n#22 0x7fa1809a91f5 <unknown>","time":{"start":1753517518423,"stop":1753517523698,"duration":5275}},{"uid":"8c46a02190ba24eb","status":"passed","time":{"start":1753516310917,"stop":1753516318602,"duration":7685}},{"uid":"22cc6bd55b66cf55","status":"broken","statusDetails":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/About/Contact#from-feedback\" class=\"btn btn-light mb-0 p-2 flex-shrink-0 mb-2 px-4 d-inline-flex align-items-center\">...</a> is not clickable at point (1247, 893). Other element would receive the click: <div id=\"cookie-alert\" class=\"alert alert-primary py-2 m-0 bg-primary1 border-0 rounded-0 alert-dismissible fade show text-center overflow-hidden fixed-bottom\" role=\"alert\" style=\"display: block;\">...</div>\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#elementclickinterceptedexception\nStacktrace:\n#0 0x557187da887a <unknown>\n#1 0x55718784d2e0 <unknown>\n#2 0x5571878a5c6c <unknown>\n#3 0x5571878a3ad2 <unknown>\n#4 0x5571878a1172 <unknown>\n#5 0x5571878a0890 <unknown>\n#6 0x5571878930ea <unknown>\n#7 0x5571878c48b2 <unknown>\n#8 0x557187892a6a <unknown>\n#9 0x5571878c4a7e <unknown>\n#10 0x5571878ea71c <unknown>\n#11 0x5571878c4683 <unknown>\n#12 0x557187890b5b <unknown>\n#13 0x557187891f31 <unknown>\n#14 0x557187d6d7cb <unknown>\n#15 0x557187d715d4 <unknown>\n#16 0x557187d542c9 <unknown>\n#17 0x557187d72178 <unknown>\n#18 0x557187d386bf <unknown>\n#19 0x557187d95e78 <unknown>\n#20 0x557187d96056 <unknown>\n#21 0x557187da7b96 <unknown>\n#22 0x7fbfac9251f5 <unknown>","time":{"start":1753514139474,"stop":1753514146218,"duration":6744}},{"uid":"c8e2f3d148a62710","status":"passed","time":{"start":1753513066866,"stop":1753513073594,"duration":6728}},{"uid":"16810ea24f4d9688","status":"passed","time":{"start":1753508160488,"stop":1753508167058,"duration":6570}},{"uid":"f37a1ccaf540eee4","status":"passed","time":{"start":1753506513060,"stop":1753506519187,"duration":6127}},{"uid":"37c048e68ca25e5d","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div/p/a[@href='/About/Contact#from-feedback']\"}\n  (Session info: chrome=138.0.7204.168); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#nosuchelementexception\nStacktrace:\n#0 0x564a8ad8970e <unknown>\n#1 0x564a8a825a51 <unknown>\n#2 0x564a8a87060f <unknown>\n#3 0x564a8a8707e1 <unknown>\n#4 0x564a8a8b5694 <unknown>\n#5 0x564a8a89401d <unknown>\n#6 0x564a8a8b3444 <unknown>\n#7 0x564a8a893dc3 <unknown>\n#8 0x564a8a864959 <unknown>\n#9 0x564a8a8654c1 <unknown>\n#10 0x564a8ad558aa <unknown>\n#11 0x564a8ad58c11 <unknown>\n#12 0x564a8ad586a8 <unknown>\n#13 0x564a8ad590b5 <unknown>\n#14 0x564a8ad3f34c <unknown>\n#15 0x564a8ad59421 <unknown>\n#16 0x564a8ad27e2a <unknown>\n#17 0x564a8ad77855 <unknown>\n#18 0x564a8ad77a4e <unknown>\n#19 0x564a8ad889f5 <unknown>\n#20 0x7f68f133e1f5 <unknown>","time":{"start":1753455869429,"stop":1753455886411,"duration":16982}},{"uid":"9575faa0157e7a1b","status":"failed","statusDetails":"AssertionError: URL не соответствует странице отправки фидбека\nassert 'https://excu...from-feedback' == 'https://excu...from-feedback'\n  \n  - https://excursium.com/About/Contact#from-feedback\n  ?                            ^\n  + https://excursium.com/Verify/Entry?returnUrl=%2FAbout%2FContact#from-feedback\n  ?                       ++++++++++++++++++++++++++     ^^^","time":{"start":1753454933846,"stop":1753454946404,"duration":12558}},{"uid":"e0ebf72e6f739679","status":"passed","time":{"start":1753454470646,"stop":1753454476972,"duration":6326}},{"uid":"4305d4688470106d","status":"passed","time":{"start":1753454332937,"stop":1753454339021,"duration":6084}},{"uid":"5c897054090805de","status":"passed","time":{"start":1753454195572,"stop":1753454204011,"duration":8439}},{"uid":"86b67ba113d7f56b","status":"passed","time":{"start":1753453916242,"stop":1753453924483,"duration":8241}},{"uid":"ece1fcfc1151a95b","status":"passed","time":{"start":1753453581541,"stop":1753453590608,"duration":9067}},{"uid":"204b22282c123799","status":"passed","time":{"start":1753453176407,"stop":1753453183377,"duration":6970}},{"uid":"4cd2ef4b70222c16","status":"passed","time":{"start":1753451768089,"stop":1753451778318,"duration":10229}}]},"tags":["user_case"]},"source":"d6afe49143cac8db.json","parameterValues":[]}